fcrepo-clients/src/main/java/edu/unc/lib/dl/fcrepo4/Repository.java

```diff
@@ -213,6 +245,27 @@ public Model getObjectModel(URI metadataUri) throws FedoraException {
 		}
 	}
 
+	/**
+	 * 
+	 * @param subject
+	 * @param property
+	 * @param object
+	 */
+	public void createRelationship(PID subject, Property property, Resource object) {
+		String sparqlUpdate = RDFModelUtil.createSparqlInsert(subject.getRepositoryPath(), property, object);
```

Comment:
Just out of curiosity, how doe getRepositoryPath() and getRepositoryURI(), which is used a few lines down, differ?